Source/Platform/DefaultOpenTK/Backend/Graphics/NativeWindow.cs

```diff
@@ -274,5 +275,30 @@ private void OnRenderFrame(FrameEventArgs e)
 			Profile.TimeSwapBuffers.EndMeasure();
 			Profile.TimeRender.EndMeasure();
 		}
+
+		public void SetCursor(ContentRef<Pixmap> cursor, Point2 hotspot)
+		{
+			if (!cursor.IsAvailable)
+			{
+				Logs.Core.WriteWarning("SetCursor: pixmap {0} not available", cursor.Name);
+			}
```

Comment:
I think it makes more sense here to _not_ check if `cursor` is available - hear me out:

If cursor is null, or unavailable, this is always an error, there is no way it could ever be valid input for this method. The appropriate way to deal with it would be to throw an `ArgumentNullException` in some initial validation. I'm not a fan of adding validation everywhere, _but_ when you look at the method implementation, it will actually just throw a `NullReferenceException` if `cursor` is null. Which is reasonably close. A warning on the other hand would just mask an actual error on the developer side with good intentions. To summarize:

- ❌ Warning and early-out: Unnecessarily masks a developer error.
- ✔️ ArgumentNullException: Explicit validation, ensure method is used as intended.
- ✔️ Not doing anything: Will cause an exception anyway, but without added code - personally, I'd be fine with that, though I can't argue best practices here.

-----

_Edit:_ On second thought, let's forget what I wrote above. Maybe specifying `null` should actually be valid input and mean "reset cursor to OS cursor". Can we do that?

Reply:
Done